src/util/formatNumber.ts

```diff
@@ -1,3 +1,3 @@
 /** Adds commas to a number. */
 // TODO: Localize
-export const formatNumber = (n: number, locale='en-US') => n.toLocaleString(locale)
+export const formatNumber = (n: number, locale='en-US') => n.toLocaleString(locale)
```

Comment:
Restore newline at end of file.

Do you have `EditorConfig` configured in your editor? Usually that will automatically enforce this.